modules/nf-core/gatk4/filterintervals/meta.yml

```diff
@@ -0,0 +1,53 @@
+name: "gatk4_filterintervals"
+description: Filters intervals based on annotations and/or count statistics.
+keywords:
+  - gatk4
+  - gatk4_filterintervals
+tools:
+  - "gatk4":
+      description:
+        Developed in the Data Sciences Platform at the Broad Institute, the toolkit offers a wide variety of tools
+        with a primary focus on variant discovery and genotyping. Its powerful processing engine
+        and high-performance computing features make it capable of taking on projects of any size.
+      homepage: https://gatk.broadinstitute.org/hc/en-us
+      documentation: https://gatk.broadinstitute.org/hc/en-us/categories/360002369672s
+      doi: "10.1158/1538-7445.AM2017-3590"
+      licence: ["Apache-2.0"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - input:
+      type: file
+      description: Read counts input file
+      pattern: "*.{tsv, hdf5, interval_list}"
+  - intervals:
+      type: file
+      description: Processed interval list file (processed_intervals.interval_list)
+      pattern: "*.{bed, interval_list}"
```

Comment:
bed ;)

Reply:
Yep you're right. In this module it only takes interval_list file.